pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -53,6 +53,8 @@ const (
 	localStorageStateFile          = "graceful_node_shutdown_state"
 )
 
+var podDefaultTerminationPeriod int64 = 30
```

Comment:
Can you get this value from the pod API rather than defining it here also?

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2977

Reply:
Fixed. Thanks.